pep8.py

```diff
@@ -231,6 +231,22 @@ def maximum_line_length(physical_line, max_line_length, multiline):
 ##############################################################################
 # Plugins (check functions) for logical lines
 ##############################################################################
+def triple_quotes(logical_line, tokens):
+    r"""Triple quotes should use double quote characters.
+    Okay: s = ""\"a""\"
+    Okay: s = r""\"a""\"
+    E999: s = '''a'''
+    E999: s = r'''a'''
+    """
+    triple = "'''"
+    for token_type, text, start, end, line in tokens:
+        text = remove_string_prefix(text)
+        if token_type == tokenize.STRING and \
```

Comment:
Please follow [the project convention](https://github.com/SylvainDe/pep8/blob/3761020583a93819c43dca65ccfcfa30d65ce93e/pep8.py#L534..L536). Do not use `\` for continuation.